keps/sig-instrumentation/3466-kubernetes-component-health-slis/README.md

```diff
@@ -337,7 +337,7 @@ We will target this feature behind a flag `ComponentSLIs`
 ###### How can this feature be enabled / disabled in a live cluster?
 
 
-- [ ] Feature gate (also fill in values in `kep.yaml`)
+- [X] Feature gate (also fill in values in `kep.yaml`)
```

Comment:
L364: can you please ensure this is done and update this KEP to describe it before you switch feature-gate to enabled in k/k

Reply:
https://github.com/kubernetes/kubernetes/pull/114997

Reply:
This doesn't add enablement/disablement tests. This PR add feature tests.

Given that this feature is purely in-memory (no persistent state), the enablement/disablement test (which requires switching FG anyway) aren't really needed. So I'm fine with saying something like:
`
Given this feature is purely in-memory, no enablement/disablement tests are needed.
`

But let's not pretend they actually exist :)

Reply:
Sure. Updated